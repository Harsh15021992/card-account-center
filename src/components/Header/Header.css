.header {
  height: 50px;
  background: #eaeaea;
}
.header-wrapper {
  width: 80%;
  margin: 0 auto;
}
.logo-container {
  display: inline-block;
  width: auto;
  font-size: 1.5em;
  color: darkslategray;
  margin-top: 8px;
}
