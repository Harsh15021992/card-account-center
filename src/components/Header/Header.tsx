import React from "react";
import "./Header.css";

const Header: React.FC = () => {
  return (
    <header className="header">
      <div className="header-wrapper">
        <p className="logo-container">Bank Logo</p>
      </div>
    </header>
  );
};

export default Header;
