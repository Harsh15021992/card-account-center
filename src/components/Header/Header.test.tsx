import React from "react";
import Header from "./Header";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";

configure({ adapter: new Adapter() });

describe("Header Component Test", () => {
  const tree = shallow(<Header />);
  it("To match Snapshot", () => {
    expect(tree).toMatchSnapshot();
  });
  it("should render header component text", () => {
    const container = tree.find(".logo-container");
    expect(container.length).toBe(1);
  });
});
