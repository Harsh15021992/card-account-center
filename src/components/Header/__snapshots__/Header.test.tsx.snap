// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Test To match Snapshot 1`] = `ShallowWrapper {}`;
